scripts/suite.py

```diff
@@ -49,7 +51,7 @@
                               [default: basic]
   -t <branch>, --teuthology-branch <branch>
                               The teuthology branch to run against.
-                              [default: master]
+                              [default: {default_teuthology_branch}]
```

Comment:
Maybe instead of silently changing the default in the output, we should note that it's being overridden by an environment variable?

Reply:
> Maybe instead of silently changing the default in the output, we should note that it's being overridden by an environment variable?

this looks like an overkill for this feature, because it will require more complicated code as for me, and what will be the benefit? from other side, I think it is a noise